skins/oasis/css/core/breakpoints-background.scss

```diff
@@ -0,0 +1,160 @@
+@import "skins/shared/color";
+@import "breakpoints-layout";
+@import "breakpoints-variables";
+@import "skins/shared/mixins/calc";
+
+$background-height: get_command_line_param( "background-image-height", 0 );
+$background-width: get_command_line_param( "background-image-width", 0 );
+
+@if unitless($background-height) {
+	$background-height : $background-height * 1px;
+}
+@if unitless($background-width) {
+	$background-width : $background-width * 1px;
+}
+$width-gradient: 20px;
+
+$background-width-half: $background-width / 2;
```

Comment:
It's not needed - browsers are fine with 0.5px - which should be really good if browser supports retina properly.
Similar code is in https://github.com/Wikia/app/blob/dev/skins/oasis/css/core/background.scss and worked fine for at least a year. 
Do you still want to change it?